examples/5.fat_sat_low_field/CMRA_signal_dynamics.jl

```diff
@@ -15,6 +15,7 @@ obj = cardiac_phantom(0)
 sim_method = BlochDict(save_Mz=true)
 sim_params = Dict{String,Any}("return_type"=>"mat", "sim_method"=>sim_method, "gpu"=>false,
                              "Nthreads"=>1, "Δt_rf"=>2e-4)
+sampling_params = Dict{String,Any}("Δt"=>sim_params["Δt"], "Δt_rf"=>sim_params["Δt_rf"])
```

Comment:
```suggestion
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/72790d5e927932b0a83b571459e14a4602156038